clang/test/CodeGen/aarch64-bitint-argpass.c

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: arm-registered-target
+// RUN: %clang_cc1 -triple aarch64-none-elf \
+// RUN:   -O2 \
+// RUN:   -emit-llvm -fexperimental-max-bitint-width=1024 -o - %s | FileCheck %s
```

Comment:
Can we integrate this into some existing test file for aarch64 calling conventions?

Reply:
I don't think such test file exists for aarch64. I tried searching for smth like that in CodeGen, couldn't find any file which would be suitable for this test.

Reply:
Thank you for pointing me to this ! I didn't expect there would a general tests for all ABIs ! I have adjusted the test so it correctly checks for sizes >128. Probably better fix would be to correctly set __BITINT_MAXWIDTH__  in AArch64 target, but that is story for separate patch.